spec/rails_helper.rb

```diff
@@ -421,6 +421,8 @@ class << example.exception
       end
     end
 
+    Capybara.reset_sessions!
+    Capybara.use_default_driver
```

Comment:
Weird that we have to do this

Reply:
We might be switching between mobile and desktop drivers so this is still good to have.